core/src/main/java/edu/wpi/grip/core/events/OperationAddedEvent.java

```diff
@@ -14,9 +16,11 @@
      * @param operation The operation being added
      */
     public OperationAddedEvent(Operation operation) {
```

Comment:
Why are you modifying all of these classes that have nothing to do with this PR?